Has/src/main/java/has/Task/Task.java

```diff
@@ -23,5 +25,11 @@
     private String startTime;
     private String finishTime;
     private Integer priority;
+    private String assigner;
+
+    @NotNull
```

Comment:
assignee has annotation notNull but i don't see any update on data.sql.. Actually there is no insert query for Tasks o_O 
Add some data